### PR TITLE
fixed bug for delete vertex without edge

### PR DESCRIPTION
**Pull request by monadbobo**
### What changes were proposed in this pull request?

When deleting a vertex, if the edge does not exist, then we should continue to delete the vertex instead of reporting an error.

### CODE REVIEW

**Comment by monadbobo** [comment]
depend #1983 close #1996

File: src/graph/DeleteVerticesExecutor.cpp [hunk 1]
```diff
@@ -111,20 +111,18 @@ void DeleteVerticesExecutor::execute() {
...
             std::unordered_map<EdgeType, std::shared_ptr<ResultSchemaProvider>> edgeSchema;
             auto *eschema = response.get_edge_schema();
-            if (eschema != nullptr) {
-                std::transform(eschema->cbegin(), eschema->cend(),
-                               std::inserter(edgeSchema, edgeSchema.begin()), [](auto &schema) {
-                                   return std::make_pair(
-                                       schema.first,
-                                       std::make_shared<ResultSchemaProvider>(schema.second));
-                               });
+            if (eschema == nullptr || eschema->empty()) {
```
Comment (dangleptr) [inline src/graph/DeleteVerticesExecutor.cpp]: Well done.  Could you give a UT for it.

File: src/graph/DeleteVerticesExecutor.cpp [hunk 2]
```diff
@@ -111,20 +111,18 @@ void DeleteVerticesExecutor::execute() {
...
+                continue;
             }
 
-            if (edgeSchema.empty()) {
```
Comment (darionyaphet) [inline src/graph/DeleteVerticesExecutor.cpp]: When the edge not exist, it's not a bug 👍

**Comment by darionyaphet** [comment]
Generally LGTM

**Pull request merged**